landing.css

```diff
@@ -0,0 +1,137 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    background-color: rgb(233, 231, 231);
+}
+
+#header{
+    display: flex;
+    justify-content: space-between;
+    padding: 8px;
+}
+
+.header-title{
+    font-size: 14px;
+}
+
+.icon-title img{
+    width: 300px;
+}
+
+.menu-links{
+    display: flex;
+    list-style: none;
+    justify-content: space-between;
+}
+
+a{
+    text-decoration: none;
+    color: black;
+    margin-right: 32px;
+}
+
+form{
+    /* background-color: red; */
+    width: 50%;
+    margin: 4px auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+form h3{
+    text-align: center;
+    margin: 12px 0;
+    
+}
+
+input{
+    width: 30%;
+    margin: 6px auto;
+    padding: 4px;
+}
+
+form button{
+    margin: 12px 0;
+    background-color: goldenrod;
+    font-weight: 700;
+    padding: 4px;
+    border: none;
+}
+
+
+.banner{
+    
+    width: 50%;
+    margin: 6px auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 400px;
+}
+
+
+.fas{
+    color: goldenrod;
+}
+
+
+.icon-description{
+    display: flex;
+    justify-content: space-between;
+}
+
+
+.description{
+    margin-left: 16px;
+}
+
+.video{
+    width: 500px;
+    margin: 12px auto;
+}
+
+.cards{
+    width: 70%;
+    margin: 12px auto;
+    display: flex;
+    justify-content: space-between;
+
+}
+
+.card{
+    width: 300px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 300px;
+    color: black;
+    justify-content: space-between;
+    border: 1px solid black;
+
+}
+
+.card h5{
+    background-color: grey;
+    width: 100%;
+    text-align: center;
+}
+
+.card button{
```

Comment:
I would suggest that you use proper naming procedures for your classes. This will help other code users or partners to understand your code.